server/router/api.go

```diff
@@ -145,8 +145,8 @@ func apiRoutes(e *gin.Engine) {
 		{
 			queue.Use(session.MustAdmin())
 			queue.GET("/info", api.GetQueueInfo)
-			queue.GET("/pause", api.PauseQueue)
-			queue.GET("/resume", api.ResumeQueue)
+			queue.POST("/pause", api.PauseQueue)
```

Comment:
is breaking but :+1:

Reply:
I will add a note to the migration list

Reply:
need to adjust woodpecker-go too

Reply:
they are not used there so its fine & thanks for the docs change

Reply:
seems to be only having the info endpoint atm